Material.Styles/Themes/MaterialThemeBase.cs

```diff
@@ -39,34 +39,71 @@ public MaterialThemeBase(IServiceProvider serviceProvider)
 
         private IResourceDictionary LoadedResourceDictionary => (Loaded as Avalonia.Styling.Styles)!.Resources;
 
-        public static readonly DirectProperty<MaterialThemeBase, ITheme> CurrentThemeProperty =
-            AvaloniaProperty.RegisterDirect<MaterialThemeBase, ITheme>(
+        public static readonly DirectProperty<MaterialThemeBase, ITheme?> CurrentThemeProperty =
+            AvaloniaProperty.RegisterDirect<MaterialThemeBase, ITheme?>(
                 nameof(CurrentTheme),
                 o => o.CurrentTheme,
                 (o, v) => o.CurrentTheme = v);
 
-        private ThemeStruct _currentTheme = new();
+        private ThemeStruct? _currentTheme;
 
         /// <summary>
         /// Get or set current applied theme
         /// </summary>
         /// <returns>
         /// Returns a STRUCT implementing ITheme interface 
         /// </returns>
-        public ITheme CurrentTheme {
-            get => new ThemeStruct(_currentTheme);
+        public ITheme? CurrentTheme {
+            get
+            {
+                if (!_currentTheme.HasValue)
+                    return null;
+                
+                return new ThemeStruct(_currentTheme.Value);
+            }
             set {
+                if (!_currentTheme.HasValue && value == null)
+                    return;
+                
                 var oldTheme = _currentTheme;
-                var newTheme = new ThemeStruct(value);
 
-                if (EqualityComparer<ITheme>.Default.Equals(oldTheme, newTheme)) return;
-                _currentTheme = newTheme;
-                RaisePropertyChanged(CurrentThemeProperty, oldTheme, newTheme);
-                StartUpdatingTheme(oldTheme, newTheme);
+                if (value != null)
+                {
+                    var newTheme = new ThemeStruct(value);
+
+                    if (oldTheme.HasValue && EqualityComparer<ITheme>.Default.Equals(oldTheme, newTheme))
+                        return;
+                
+                    _currentTheme = newTheme;
+                    RaisePropertyChanged(CurrentThemeProperty, oldTheme, newTheme);
+                    StartUpdatingTheme(oldTheme, newTheme);
+                }
+                else
+                {
+                    _currentTheme = null;
+                    RaisePropertyChanged(CurrentThemeProperty, oldTheme, null);
+                    StartUpdatingTheme(oldTheme, null);
```

Comment:
We can't assign `null` to `CurrentTheme`, cuz it cause `NullReferenceException` here. We don't have to support null here. Or, at the very least, I can't find a use for it.